src/components/InventoryGroups/utils/api.js

```diff
@@ -25,6 +25,17 @@ export const validateGroupName = (name) => {
     .then((resp) => resp?.results.some((group) => group.name === name));
 };
 
+export const updateGroupById = (id, payload) => {
+    return instance.put(`${INVENTORY_API_BASE}/groups/${id}`, {
+        Name: payload.name,
+        Type: 'static'
+    });
+};
+
```

Comment:
I believe it's not needed in this PR.
```suggestion
```

Reply:
You are correct I added this to evade future conflicts in this file. But I can remove it :)